xblock/field_data.py

```diff
@@ -148,7 +148,12 @@ def _field_data(self, block, name):
         if scope not in self._scope_mappings:
             raise InvalidScopeError(scope)
```

Comment:
Isn't this line trying to do something similar? Maybe you can consolidate these two conditions using `self._scope_mappings.get(...)`

Reply:
Done: e048432336cad94554a4096edc6f42a23d8d7969